pkg/network/encoding/format.go

```diff
@@ -301,6 +308,31 @@ func formatDNSStatsByDomain(stats map[string]map[network.QueryType]network.DNSSt
 	}
 	return m
 }
+func formatSingleADNSStatsByDomain(stats map[string]map[network.QueryType]network.DNSStats, domainSet map[string]int) map[int32]*model.DNSStats {
+	m := make(map[int32]*model.DNSStats)
+	for d, bytype := range stats {
+
+		for t, stat := range bytype {
+			if t != network.DNSTypeA {
```

Comment:
I think we can remove this check. If they have configured to capture A and AAAA, but breaking stats down by query type is disabled, then we can include all stats.